platform-sdk/platform-apps/tests/PlatformTestingTool/src/test/java/com/swirlds/demo/platform/MapValueSerializableTest.java

```diff
@@ -163,7 +161,6 @@ private void insertIntoFCQMap(
     }
 
     @Test
-    @Tag(TIME_CONSUMING)
```

Comment:
> kept in test but the tag is removed (which were just a few). The rational behind this is, that these tags were added a long time ago to single out long running tests. This mechanism was not kept and the longest tests in test are not event marked with TIME_CONSUMING (e.g. [NodeCreateHandlerTest](https://scans.gradle.com/s/lrdm5ghb5yz66/tests/task/:app-service-addressbook-impl:test/details/com.hedera.node.app.service.addressbook.impl.test.handlers.NodeCreateHandlerTest)). Also, with tests running in parallel now, removing the longest running tests is not the only strategy to get the total time down. What we do here will be reevaluated in the next step.

As I wrote in the description I removed the tag for tests that are not timing sensitive. Because ther are currently tests that are more time consuming which are not marked. Basically the tag was added months ago for some tests, but afterwards this was not maintained.

We'll reintroduce "time consuming" as a separate category (with a separate source.folder) after this and then need to evaluate which tests are moved there.

Reply:
If my memory serves, some of these tests were marked timing sensitive, not due to length of time, but to prevent concurrency as there were static interactions if the tests were run in parallel in the same JVM.   I don't know that this applied in this case, but I believe I remember us marking tests with the annotation when we discovered this problem.   It could be that the code has been fixed as we've tried to make things deterministic and non-static for TURTLE.  @poulok , @litt3,  @lpetrovic05, do you recall anything about this?

Reply:
I was mistaken.  We were just using the TIMING_SENSITIVE tag.